### DISCUSSION TITLE
Use postgres database with docker image

### DISCUSSION THREAD

**Discussion by nitanmarcel** [toeverything/AFFiNE#4955] [Q&A]
I want to host AFFiNE so I can have some notes and share it with others. It will be personal use with only me being able to create notes.

The biggest issue I have now is storage, I have a small VPS from hetzer for just 1$ that has only 20GB storage from which I only have left like 2, and I'm hanging as hard as I can on it. Can I host the docker image and save everything on a database? If yes can someone point me in the right direction?

I have to use docker so I can build everything locally and just pull it to the server since `yarn` takes ages to successfully install everyting, resulting in my VPS being frozen most of the time.

Thanks.

**Comment by bolavefasfas** [comment]
Bro the docker image is not updated for long time 
idk the eta but hoping it might be soon idk

**Comment by EYHN** [comment]
As @bolavefasfas said the self-host docker image has not been updated for a long time and we are developing a new self-host version, The details in this regard are still under consideration and planning, I think by then we will have a solution that can be deployed on cheap vps.

Appreciate your patience and stay tuned for updates!

Related: https://github.com/toeverything/AFFiNE/issues/4914

**Comment by almereyda** [comment]
It was trivial to `pull` and `up` the current Compose manifest.

- https://github.com/toeverything/AFFiNE/blob/b15ae21c45cd1c2577f673854a862059a0c8eb95/.github/deployment/self-host/compose.yaml

This uses Postgres by default and was updated to work three days ago.

https://github.com/toeverything/AFFiNE/blob/b15ae21c45cd1c2577f673854a862059a0c8eb95/.github/deployment/self-host/compose.yaml#L44-L59

Maybe others can reproduce this, and this conversation can be marked as answered?
